services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -138,14 +136,14 @@ async def send_mail(app: web.Application, msg: MIMEText):
         if cfg.SMTP_TLS_ENABLED:
             log.info("Starting TLS ...")
             await smtp.starttls(validate_certs=False)
-        if cfg.SMTP_USERNAME:
+        if cfg.SMTP_USERNAME and cfg.SMTP_PASSWORD:
```

Comment:
BTW ensure SMTP_USERNAME and SMTP_PASSWORD do NOT validate against empty strings

Reply:
If the values are empty strings they are treated as false values. This check will be false.